### ISSUE TITLE
Arrow in gutter is not draggable.

### ISSUE THREAD

**Issue by ig13** [igoriakovlev/JumpToLine#1]
When I run in debug mode, the orange arrow appears in the gutter, but nothig happens when I try to drag it with mouse or click on it with Ctrl pressed.

My setup: Jump to Line 0.1.0, IDEA 2020.2.1 CE (build #IC-202.6948.69), Windows 8.1 6.3.

Also I checked idea.log - can't see nothing suspicious there. The only line related to the plugin is "Loaded custom plugins: ..., Jump to Line (0.1.0), ...".

**Comment by igoriakovlev** [comment]
Hi there,

Was it the only debug session? Is it reproducible? If so, could you give me an example project and steps to reproduce? It is too little info to make any assumptions for now.

**Comment by ig13** [comment]
> Was it the only debug session? 

If you mean running IDEA or plugin in debug mode then no. I talked about normal plugin usage.

> Is it reproducible? If so, could you give me an example project and steps to reproduce? 

To the moment I have found out that described behaviour is kinda project-dependent, indeed. In some projects dragging works, but the plugin still does not work properly though, since I always get "An error occured while popping stack frame: information about the frame is not available" ([example](https://imgur.com/a/JKz6qi3), JIC).

I cannot provide the original project, sorry.

**Comment by igoriakovlev** [comment]
Maybe it worth to send me IDEA logs (if it does not have any sensetive data of course).
(https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files)

**Comment by Gravityzwell** [comment]
For what it's worth, my tests with Kotlin and Idea latest version on Windows worked without issue.

**Comment by chylex** [comment]
I ran into this issue too, but it was because I was testing the plugin in a main method which is apparently not supported. It would be helpful if clicking the 'Skip to Cursor' action showed you the reason why you cannot jump, currently it seems to just do nothing at all if one of the preconditions fails.

**Comment by igoriakovlev** [comment]
Yes, plugin has limitations for jumps in _main_ functions and other limitations described in [here](https://blog.jetbrains.com/idea/2020/08/jump-to-any-line-while-debugging/). I've added error reporting to menu handler.
For now, You can manually build and install plugin with these changes.
